packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -387,6 +387,22 @@ export class MgtPerson extends MgtTemplatedComponent {
   })
   public personCardInteraction: PersonCardInteraction;
 
+  /**
+   * Get the scopes required for person
+   *
+   * @static
+   * @return {*}  {string[]}
+   * @memberof MgtPerson
+   */
+  public static requiredScopes: string[] = [
+    'contacts.read',
```

Comment:
`contacts.read` is only required if enabled in the global person config

Reply:
Good catch! That already justifies using getter instead of a property